biomejs/biome
# feat: i18n setup and some Japanese translation

## Summary
Partial work of #880




## Test Plan
- Check if i8n is ready to go
- Check if home and 404 pages are translated in Japanese
